### PR TITLE
Libxc

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

**Comment by certik** [comment]
It seems to fail with the error:
```
checking for libxc... no (-I/usr/include -lxcf90 -lxc)
configure: error: Could not find required libxc library.
Makefile:9: make.inc: No such file or directory
make: *** No rule to make target 'make.inc'.  Stop.
```

File: recipe/build.sh [hunk 1]
```diff
@@ -1,13 +1,12 @@
 #!/bin/bash
-./configure --prefix=${PREFIX} --with-libxc \
+./configure --prefix=${PREFIX} --with-libxc --with-libxc-prefix=${PRLEFIX}/lib \
```
Comment (certik) [inline recipe/build.sh:2]: Shouldn't this be?
```suggestion
./configure --prefix=${PREFIX} --with-libxc --with-libxc-prefix=${PREFIX}/lib \
```

**Comment by certik** [comment]
Locally in a Conda environment, I use `${CONDA_PREFIX}`, but I can't remember right now, but probably just `${PREFIX}` works in conda build and conda-forge.

**Comment by jan-janssen** [comment]
Currently I get the following error:

```
/home/conda/feedstock_root/build_artifacts/qe_1590776253077/_build_env/bin/../lib/gcc/x86_64-conda_cos6-linux-gnu/7.5.0/../../../../x86_64-conda_cos6-linux-gnu/bin/ld: ../../Modules/libqemod.a(funct.o): in function `__funct_MOD_set_dft_from_name':
funct.f90:(.text.__funct_MOD_set_dft_from_name+0x613): undefined reference to `__xc_f03_lib_m_MOD_xc_f03_func_init'
/home/conda/feedstock_root/build_artifacts/qe_1590776253077/_build_env/bin/../lib/gcc/x86_64-conda_cos6-linux-gnu/7.5.0/../../../../x86_64-conda_cos6-linux-gnu/bin/ld: funct.f90:(.text.__funct_MOD_set_dft_from_name+0x61c): undefined reference to `__xc_f03_lib_m_MOD_xc_f03_func_get_info'
/home/conda/feedstock_root/build_artifacts/qe_1590776253077/_build_env/bin/../lib/gcc/x86_64-conda_cos6-linux-gnu/7.5.0/../../../../x86_64-conda_cos6-linux-gnu/bin/ld: funct.f90:(.text.__funct_MOD_set_dft_from_name+0x62f): undefined reference to `__xc_f03_lib_m_MOD_xc_f03_func_info_get_n_ext_params'
/home/conda/feedstock_root/build_artifacts/qe_1590776253077/_build_env/bin/../lib/gcc/x86_64-conda_cos6-linux-gnu/7.5.0/../../../../x86_64-conda_cos6-linux-gnu/bin/ld: funct.f90:(.text.__funct_MOD_set_dft_from_name+0x63a): undefined reference to `__xc_f03_lib_m_MOD_xc_f03_func_end'
collect2: error: ld returned 1 exit status
```

In addition:

> Note that currently only a (small) subset of functionals implemented in libxc can be used and that the libxc-enabled version cannot use functionals from Quantum ESPRESSO. 

So I close this pull request for now.

**Comment by certik** [comment]
@jan-janssen thanks for trying out anyway. As you say, it does not seem to be well supported, so better not to depend on libxc for now.

**Pull request closed without merging**